@W-21216935 | Documentation for AS2 v7.0.0 (#3143)
mulesoft/docs-connectors: as2/7.0/modules/ROOT/pages/as2-connector-studio.adoc

. On the *Authentication* tab, configure the authentication method for the outbound request.
+
You can reference a configuration file that contains ANT-style property placeholders (recommended),
or you can enter your authorization credentials in the global configuration properties.
For information about the benefits of using property placeholders and how to configure them, see xref:connectors::introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration].
. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
. Click *Test Connection* to confirm that Mule can connect with the specified server.
. Click *OK*.

[[configure-external-libraries]]
== Configure a Global Element for External Libraries

Mule applications using AS2 Connector must contain only one global element configuration to provide the external libraries required for the selected security mode.

To configure the global element for AS2 Connector and add external libraries, follow these steps:

> . In the *Package Explorer* or *Canvas* in Anypoint Studio, open your Mule flow or configuration.

Reviewer: ```suggestion
. In *Package Explorer* or Anypoint Studio canvas, open your Mule flow or configuration.
```

. Click *Global Elements*. Or click *Create global element*, and then click the plus icon next to Global Elements.

> . Click *Create*, and then search for the *AS2* or *External Libs* option and select `AS2 Connector External Libs - Non FIPS 140-3`.

Reviewer: ```suggestion
. Click *Create*, and then search for the `AS2` or `External Libs` option and select *AS2 Connector External Libs - Non FIPS 140-3*.
```

. Name the configuration.

> . Click *Configure* > *Add recommended libraries* to add the libraries.

Reviewer: ```suggestion
. Click *Configure* > *Add recommended libraries*.
```

. Click *OK* to include the libraries as dependencies and shared libraries in the Mule application `pom.xml` file.

> . Verify that your project has the required dependencies for your security mode. Add the dependencies via Maven if they haven't been added automatically.

Reviewer: ```suggestion
. Verify that your project has the required dependencies for your security mode. 
+
Add the dependencies via Maven if they aren't added automatically.
```

. Click *Test Connection* to confirm that Mule can connect with the specified server.
. Click *OK*.

[[configure-external-libraries]]

> == Configure a global element to add the required external libraries

Reviewer 2: Configure a Global Element for External Libraries

> When you use as2 connector in the app, there should only be one global connector configuration to add the external libraries needed for the specific security mode. 

Reviewer 2: Mule applications using AS2 Connector must contain only one Global Connector Configuration to provide the external libraries required for the selected security mode.

To configure the global element for AS2 Connector and add external libraries, follow these steps:

> . Select the *Package Explorer* or *Canvas*, open your Mule flow or configuration.

Reviewer 2: Select the *Package Explorer* or *Canvas* to open your Mule flow or configuration.

Reviewer 2: Package Explorer and Canvas should be in bold and not italics

> . Click *Global Elements* (or *Create global element* / the plus icon next to Global Elements).

Reviewer 2: I reviewed a similar section on the upgrade file, so those changes can be applied here.
